chevah/python-package
# [#5048] Stop using PyCryptodome wheel in RHEL 6.

Scope
=====

Our issue with pycryptodome on RHEL 6 was fixed upstream. For details see ​https://github.com/Legrandin/pycryptodome/issues/187#issuecomment-412305122.

A new version, 3.6.5, was released with this fix. To be tested.

High priority, to keep exchanges with upstream on the fast lane.


Changes
=======

Updated used pycryptodome to 3.6.5.

Stop using upstream wheel in RHEL 6.


How to try and test the changes
===============================

reviewers: @adiroiban 

Please double-check changes for `chevah_build.sh` in https://github.com/chevah/python-package/pull/113/commits/200570b0b4e44813030e180c504c7d4cebe1758b.

Run the automated tests.
